### ISSUE TITLE
Add ssz-js

### ISSUE THREAD

**Issue by prestonvanloon** [prysmaticlabs/prysm-testnet-site#37]
We need a way to decode the deposit data in the browser so that we know the public key of the deposit data.

This is currently blocked by https://github.com/ChainSafe/ssz-js/issues/81 and the use of Node.js `Buffer` within the library.

**Comment by prestonvanloon** [comment]
No longer needed with new deposit contract ABI.
